index.html

```diff
@@ -650,21 +654,37 @@ <h3>The <dfn data-dfn-for="GlobalEventHandlers" data-dfn-type="event">lostpointe
+                  <a data-cite="uievents/#event-type-auxclick">auxclick</a> and <a data-cite="uievents/#event-type-contextmenu">contextmenu</a>
+                  events defined in [[UIEVENTS]].  These events are typically tied to user interface activation, and are fired even from non-pointer input devices, such as keyboards.</p>
+                <p>These events MUST be of type <code>PointerEvent</code>, and are subject to the additional requirements mentioned in the rest of this section.</p>
+                <section>
+                  <h4>Attributes (other than coordinates)</h4>
+                  <p>For these events, all <code>PointerEvent</code> specific attributes (defined in this spec) other than <code>pointerId</code> and <code>pointerType</code> MUST have their default values. In addition:</p>
+                  <ul>
+                    <li>If the events are generated by a pointing device, their <code>pointerId</code> and <code>pointerType</code> MUST be the same as the PointerEvents that caused these events.</li>
+                    <li>If the events are generated by a non-pointing device (such as voice recognition software or a keyboard interaction), <code>pointerId</code> MUST be <code>-1</code> and <code>pointerType</code> MUST be an empty string.</li>
+                  </ul>
+                </section>
+                <section>
+                  <h4>Event coordinates</h4>
+                  <p>As noted in {{PointerEvent}}, [[[CSSOM-VIEW]]] proposes to redefine the various coordinate properties (<code>screenX</code>, <code>screenY</code>, <code>pageX</code>, <code>pageY</code>, <code>clientX</code>,
+                      <code>clientY</code>, <code>x</code>, <code>y</code>, <code>offsetX</code>, <code>offsetY</code>) as <code>double</code>, to allow for fractional coordinates.
+                      However, this change — when applied only to {{PointerEvent}}, but not to regular {{MouseEvent}} — has proven to lead to web compatibility issues with legacy code
+                      in the case of <code>click</code>, <code>auxclick</code>, and <code>contextmenu</code>. For this reason, user agents that have implemented the proposed
+                      change in [[[CSSOM-VIEW]]] only for {{PointerEvent}} MUST convert the various coordinate properties for the <code>click</code>, <code>auxclick</code>, and <code>contextmenu</code>
+                      to <code>long</code> values (as defined in the original [[[UIEVENTS]]]) using <a data-cite="ECMASCRIPT#sec-math.floor">Math.floor</a> [[ECMASCRIPT]].</p>
+                </section>
+                <section>
+                  <h4>Event dispatch</h4>
+                  <p>If a <code>click</code>, <code>auxclick</code> or <code>contextmenu</code> event <var>E</var> is fired in response to a {{GlobalEventHandlers/pointerup}} event, and
```

Comment:
Capital E looks unusual, no?

Reply:
Lowercased it, thanks.

To be frank, I wanted to avoid the first use of `<var>` in this spec but couldn't find a better way to point to "that event" both concisely and precisely.